scripts/build/Platform/Linux/destroy_cdk_applications.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+
+# Deploy the CDK applications for AWS gems (Linux only)
+# Prerequisites:
+# 1) Node.js is installed
+# 2) Node.js version >= 10.13.0, except for versions 13.0.0 - 13.6.0. A version in active long-term support is recommended.
+
+SOURCE_DIRECTORY=$(dirname "$0")
+PATH=$SOURCE_DIRECTORY/python:$PATH
+GEM_DIRECTORY=$SOURCE_DIRECTORY/Gems
+
+DestroyCDKApplication()
+{
+# Destroy the CDK application for a specific AWS gem
+GEM_NAME=$1
+echo [cdk_destruction] Destroy the CDK application for the $GEM_NAME gem
+pushd $GEM_DIRECTORY/$GEM_NAME/cdk
+
+# Revert the CDK application code to a stable state using the provided commit ID
+if ! git checkout $COMMIT_ID -- .;
+then
+    echo [git_checkout] Failed to checkout the CDK application for the %GEM_NAME% gem using commit ID $COMMIT_ID
```

Comment:
should be `$GEM_NAME` (not Windows-y `%GEM_NAME%`)

Reply:
Nice catch! Will make the change.